src/notification/index.js

```diff
@@ -79,10 +79,8 @@ export const getAccountFromNotificationData = (
       realm_uri,
       parsed_url: realmUrl,
       match_count: urlMatches.length,
-      unique_identities_count: new Set(urlMatches.map(matchIndex => identities[matchIndex].email))
-        .size,
+      unique_accounts_count: new Set(urlMatches.map(matchIndex => accounts[matchIndex].email)).size,
```

Comment:
Best to keep the name for this one commit. :slightly_smiling_face: 
* The commit is marked NFC, which it isn't if a name sent in logging changes.
* "Unique identities" is still an accurate description, and clearer what it means, because we're only counting up distinct emails (for this realm), not looking for other properties of an `Account`.